web/styles/left_sidebar.css

```diff
@@ -395,6 +396,35 @@
         opacity: var(--opacity-sidebar-heading-icon);
         justify-self: center;
     }
+
+    &.hide_unread_counts {
+        .unread_count,
+        .unread_count.hide,
+        .masked_unread_count {
+            display: none;
+        }
+
+        /* When an empty unread count is hidden,
+           hide the masked unread count, too. */
+        .unread_count.hide + .masked_unread_count {
+            display: none;
+        }
+
+        .masked_unread_count {
+            display: flex;
```

Comment:
I'm pretty sure you want `display: flex` here, and so you shouldn't include it in the `display: none` rule above

Reply:
I think this is all you need, did you confirm the rest was doing something or did you just assume it was because you saw it elsewhere?

```css
    &.hide_unread_counts {
        .unread_count {
            display: none;
        }

        .masked_unread_count {
            display: flex;
        }

        &:hover {
            .unread_count {
                display: block;
            }

            .masked_unread_count {
                display: none;
            }
        }
    }
```